src/main/java/com/revature/model/Job.java

```diff
@@ -80,97 +74,83 @@ public Job(int id, String title, String description, String location, Set<Skill>
 	public int getId() {
 		return id;
 	}
-
 	/** Sets the integer that uniquely identifies this job.
 	 * @param The new value of the integer that uniquely identifies this job. */
 	public void setId(int id) {
 		this.id = id;
 	}
-
 	/** Returns the name of this job. 
 	 * @param The name of this job. */
 	public String getTitle() {
 		return title;
 	}
-
 	/** Sets the name of this job. 
 	 * @param title The new name of this job. */
 	public void setTitle(String title) {
 		this.title = title;
 	}
-
 	/** Returns a description of this job. 
 	 * @return A string describing this job.*/
 	public String getDescription() {
 		return description;
 	}
-
 	/** Sets the description of this job. Maximum length is 10,000 characters.
 	 * @param description The new description for this job. Maximum length is 10,000 characters. */
 	public void setDescription(String description) {
 		this.description = description;
 	}
-
 	/** Returns the set of skills that candidates applying to this job are expected to have. 
 	 * @return The set of skills that candidates applying to this job are expected to have. */
 	public Set<Skill> getSkills() {
 		return skills;
 	}
-
 	/** Updates the set of skills that candidates applying to this job are expected to have.
 	 * @param skills The new the set of skills that candidates applying to this job are expected to have. */
 	public void setSkills(Set<Skill> skills) {
 		this.skills = skills;
 	}
-
 	/** Returns true if the job opening is currently filled, and false otherwise. 
 	 * @return True if the job opening is currently filled, and false otherwise. */
 	public boolean isFilled() {
 		return isFilled;
 	}
-
 	/** Updates the boolean flag that marks whether the job opening is currently filled (true) or unfilled (false). 
 	 * @param isFilled The new boolean value that marks whether the job opening is currently filled (true)
 	  or unfilled (false).*/
 	public void setFilled(boolean isFilled) {
 		this.isFilled = isFilled;
 	}
-
 	/** Returns a string identifying where the job takes place. 
 	 * @return A string identifying where the job takes place. */
 	public String getLocation() {
 		return location;
 	}
-
 	/** Updates the string that identifies where the job takes place.
 	 * @param location A string that identifies where the job takes place. */
 	public void setLocation(String location) {
 		this.location = location;
 	}
-
 	/** Returns the employee that currently holds this job. Returns null if it is not held by any employee. 
 	 * @return The employee that currently holds this job. Returns null if it is not held by any employee. */
 	public Profile getProfile() {
 		return profile;
 	}
-
 	/** Sets the employee that currently holds this job. A value of null indicates that the job is unfilled.
 	 * @param profile The employee that currently holds this job. A value of null indicates that the job is unfilled. */
 	public void setProfile(Profile profile) {
 		this.profile = profile;
 	}
-
 	@Override

```

Comment:
There seemingly was no hashcode or equals method here previously, but go ahead and add them.

Reply:
Changes are made as per suggestion